public/app/features/dashboard/components/SupportSnapshot/SupportSnapshot.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { FieldType, getDefaultTimeRange, LoadingState, toDataFrame } from '@grafana/data';
+import { getPanelPlugin } from 'app/features/plugins/__mocks__/pluginMocks';
+
+import { PanelModel } from '../../state/PanelModel';
+
+import { SupportSnapshot } from './SupportSnapshot';
+
+function setup() {
+  const panel = new PanelModel({});
+  panel.plugin = getPanelPlugin({});
+  panel.getQueryRunner().setLastResult({
+    timeRange: getDefaultTimeRange(),
+    state: LoadingState.Done,
+    series: [
+      toDataFrame({
+        name: 'http_requests_total',
+        fields: [
+          { name: 'Time', type: FieldType.time, values: [1, 2, 3] },
+          { name: 'Value', type: FieldType.number, values: [11, 22, 33] },
+        ],
+      }),
+    ],
+  });
```

Comment:
would be nice with some standard mock result/values we can reuse for PanelData, do we have any?